l1-contracts/contracts/zksync/Storage.sol

```diff
@@ -143,4 +143,7 @@ struct AppStorage {
     address admin;
     /// @notice Address that the governor or admin proposed as one that will replace admin role
     address pendingAdmin;
+    /// @notice MERGE CAREFULLY: CALLOUT IF THIS IS STILL HERE IN PR:
```

Comment:
I am not totally sure we'll have native Solidity support for the blobhash by the time when the code has to be frozen. I would personally suggest using a preprocessor to hardcode the getter contract. This will be:

a) cheaper -- no need to do an additional read from storage for each commitment
b) won't pollute the storage with a variable that will deprecated in the next release

Reply:
agreed, using the preprocessor will be more elegant